NebulaClient/PacketProcessors/Factory/Entity/CreatePrebuildsRequestProcessor.cs

```diff
@@ -89,6 +86,7 @@ public void ProcessPacket(CreatePrebuildsRequest packet, NebulaConnection conn)
                 pab.waitConfirm = tmpConfirm;
                 pab.previewPose.position = tmpPos;
                 pab.previewPose.rotation = tmpRot;
+                pab.buildPreviews = tmpList;
```

Comment:
Isn't this because there are multiple players writing to the shared buildPreviews? all this code should be done under some kind of locking.

Reply:
No no, this is for leveraging the original code for creating the Prebuilds. When the client receives a packet from host to create prebuilds from previews, in short, it will do this:
```cs
tmpList = pab.buildPreviews;    //store current previews
pab.buildPreviews = packet.buildPreviews;  //set previews from packet
pab.CreatePrebuilds();     //native code to create prebuilds from previews
pab.buildPreviews = tmpList;    //restore current previews
```

Reply:
The idea in the previous PR that introduces the bug was to not restoring `buildpreviews` if the client is the author of the building. But it does not work correctly.